### PR TITLE
Create supporting resources multiple links for downloads

### PR DESCRIPTION
**Pull request by mark-pkgflow**
- Created a new resource card template partial with image and two file links
- Added supporting resources as per the trello card

image

image

image

image

### CODE REVIEW

**Comment by mark-pkgflow** [comment]
[Create supporting resources multiple links for downloads](https://trello.com/c/E3BEhzl4/388-create-supporting-resources-multiple-links-for-downloads)

File: content/workload-reduction-toolkit/address-workload-issues/behaviour-management/review-and-streamline-behaviour-management.md [hunk 1]
```diff
@@ -95,3 +95,28 @@ Provide a supportive environment for open and honest professional dialogue.
 - [Policy Exchange publication: It just grinds you down](https://policyexchange.org.uk/publication/it-just-grinds-you-down)
 
 - [Making data work: Teacher workload advisory group report](https://www.gov.uk/government/publications/teacher-workload-advisory-group-report-and-government-response)
+
+<div class="dfe-width-container govuk-grid-row">
```
Comment (AbigailMcP) [inline content/workload-reduction-toolkit/address-workload-issues/behaviour-management/review-and-streamline-behaviour-management.md:99]: This width container is making the box unaligned with the content above it on smaller screens. We can just get rid of this wrapping div. I'm also removing all these in my mobile layout PR.

```suggestion
```

**How it looks currently:**

Screenshot 2024-03-21 at 11 47 01

File: content/workload-reduction-toolkit/address-workload-issues/behaviour-management/set-up-a-centralised-system-to-record-monitor-and-reward-pupil-behaviour.md [hunk 2]
```diff
@@ -144,3 +144,28 @@ or every half-term, and used to create lists or graphs for assemblies.
 
 Some management information systems also include a messaging system that can
 flag important messages to parents and carers, form tutors or senior leaders.
+
+<div class="dfe-width-container govuk-grid-row">
```
Comment (AbigailMcP) [inline content/workload-reduction-toolkit/address-workload-issues/behaviour-management/set-up-a-centralised-system-to-record-monitor-and-reward-pupil-behaviour.md:148]: As above, remove this wrapping div.

**Review by AbigailMcP — COMMENTED** [review]
Looks good, just a couple of tweaks to html!

**Comment by mark-pkgflow** [comment]
> Looks good, just a couple of tweaks to html!

Thanks @AbigailMcP 
Updated!

**Review by AbigailMcP — APPROVED** [review]

**Comment by claire-hughez** [comment]
Spoke to Lauren and Abi, is it ok if we make the header for the supporting resource black instead of blue please Mark? Sorry, realise this wasn't part of your PR, but just think it matches better with other links/headers on the services.

On this page, where it states 'how we communicate' or 'impact graph' would be black: https://deploy-preview-102--iwwss-review.netlify.app/workload-reduction-toolkit/address-workload-issues/communications/review-and-streamline-communications/

**Comment by lauren-oseni-2** [comment]
Other than what Claire's just mentioned (re: changing the title to black), everything else looks all good :)

**Pull request merged**